### ISSUE TITLE
Would be nice to see the Release which is assigned to the Issue on the Issue page.

### ISSUE THREAD

**Issue by xeagle2** [iridakos/release_logs#25]

**Comment by iridakos** [comment]
Do you mean something like this?

![Here](http://4.bp.blogspot.com/-HsfhlQdXvE4/VRr2VAzEFPI/AAAAAAAABTs/8Rhcpbia75A/s1600/issue-release-log.png)

Isn't this already there or is it not showing to 3.x version of Redmine?

**Comment by xeagle2** [comment]
Yes, that's the same I mentioned, the info is shown if release is published only and panel doesn't show any assignment if release is still in draft status, it would be nice to show the assignment in both cases to watch if issue is assigned at least to any release.

Additionally if issue is not assigned to any release the "Release log: is visible anyway, would be better to don't display the section if there is no assignment for the issue and save space.

I like the idea of the extension, thanks for make it live!
